File: php/examples/getForGenre.php
<?php

$genre = 'Anime';
if(!empty($_GET['g'])) {
	$genre = $_GET['g'];
}

require_once '../src/BurningSeries.php';
$bs = new BurningSeries();

$series = $bs->getByGenre($genre);

echo "The following series have genre {$genre}: <br />";

foreach($series['series'] as $serie) {
	echo "{$serie['name']} (ID: {$serie['id']}) <br />";
}

File: java/examples/de/jonesboard/burningseries/examples/getForGenre.java
package de.jonesboard.burningseries.examples;

import de.jonesboard.burningseries.BurningSeries;
import de.jonesboard.burningseries.interfaces.SerieInterface;

public class getForGenre {

	public static void main(String[] args) {
		String genre = "Anime";

		BurningSeries bs = new BurningSeries();

		try {
			SerieInterface[] series = bs.getByGenre(genre);
	
			System.out.println("The following series have genre " + genre + ":");

			for(SerieInterface serie : series) {
				System.out.println(serie.getName() + " (ID: " + serie.getId() + ")");
			}
		} catch(Exception e) {
			e.printStackTrace();
		}
	}


}
